### PR TITLE
Fix xml issues

### PR DESCRIPTION
**Pull request by cloudn9**
Continuing list from #9744

### CODE REVIEW

File: docs/framework/wcf/feature-details/client-configuration.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ You can use the Windows Communication Foundation (WCF) client configuration to s
               <servicePrincipalName value="host/localhost" />  
             </identity>  
           </endpoint>  
-// Add another endpoint by adding another <endpoint> element.  
+            // Add another endpoint by adding another <endpoint> element.
```
Comment (Thraka) [inline docs/framework/wcf/feature-details/client-configuration.md:38]: Shouldn't these in this file be changed to XML comments?
```suggestion
            <!-- Add another endpoint by adding another <endpoint> element. -->
```

File: docs/framework/wcf/feature-details/client-configuration.md [hunk 2]
```diff
@@ -56,10 +56,10 @@ You can use the Windows Communication Foundation (WCF) client configuration to s
           </security>  
         </binding>  
         <binding name="Another Binding"  
-        //Configure this binding here.  
+          //Configure this binding here.
```
Comment (Thraka) [inline docs/framework/wcf/feature-details/client-configuration.md:59]:
```suggestion
          <!-- //Configure this binding here. -->
```

**Review by Thraka — CHANGES_REQUESTED** [review]

File: docs/framework/wcf/feature-details/client-configuration.md (hunk 1 above)
Comment (cloudn9) [reply docs/framework/wcf/feature-details/client-configuration.md:38]: oh yes, thank you @Thraka

**Review by Thraka — APPROVED** [review]
Awesome!!! Thanks for helping out!

**Pull request merged**